fullstack-cert/python-projects/binary-search-workshop/main.py

```diff
@@ -0,0 +1,27 @@
+def binary_search(search_list, value):
+    path_to_target = []
+    low = 0
+    high = len(search_list) - 1
+
+    while low <= high:
+        mid = (low + high) // 2
+        value_at_middle = search_list[mid]
+        path_to_target.append(value_at_middle)
+
+        if value == value_at_middle:
+            return path_to_target
+        elif value > search_list[mid]:
```

Comment:
I missed this one sorry:

```suggestion
        elif value > value_at_middle:
```